packages/apps/hr/src/components/templates/TimerListAlert/TimerListAlert.tsx

```diff
@@ -152,6 +152,7 @@ const TimerListAlert = ({
         }),
       );
     }
+    setIsAlertVisible(false);
```

Comment:
I think we should reset the dates/timesheet too
Also, the list view behind is not refresh after validation